app/screens/snack_bar/useToastToggler.tsx

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {useEffect, useState} from 'react';
+import {Navigation} from 'react-native-navigation';
+
+const useToastToggler = (componentId: string, isDismissible: boolean) => {
```

Comment:
Might not be needed... I will implement the toasts for the network connectivity as well and potentially re-used it...else I will remove it and put it back in the Snackbar screen